scripts/rebuild-ui.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+cd ../jaeger-ui
+
+if git describe --exact-match --tags $(git rev-parse HEAD) >/dev/null 2>&1; then
+    
+    # Get the release version from the tag  
+    release_version = $(git describe --exact-match --tags $(git rev-parse HEAD)) 
+
+
+    # Check if the corresponding UI release has the assets.tar.gz file uploaded
+    release_url="https://github.com/jaegertracing/jaeger-ui/releases/download/${release_version}/assets.tar.gz"
+
+    temp_file=$(mktemp)
+
+    # Download the file to the temporary file and extract it into packages/build/
+    if curl --output /dev/null --silent --head --fail "$release_url"; then
+        curl --location --output "$temp_file" "$release_url"
```

Comment:
Why do you need to check for `--head` first instead of trying to download right away? I assume it would still return an error exit code if the URL is invalid.